engine/common/rpc/convert/convert.go

```diff
@@ -927,6 +957,21 @@ func MessageToChunk(m *entities.Chunk) (*flow.Chunk, error) {
 	}, nil
 }
 
+// BlockExecutionDataEventPayloadsToJson converts all event payloads from CCF to JSON-CDC in place
+// This is a temporary workaround until a more robust solution is implemented
+func BlockExecutionDataEventPayloadsToJson(m *entities.BlockExecutionData) error {
```

Comment:
why do this in-place?

Reply:
To avoid copying the whole object. it's being done on the entity object so it won't affect the original objects